Sources/Charcoal/Filters/Inline/SegmentButton.swift

```diff
@@ -10,19 +10,19 @@ extension SegmentButton {
     }
 }
 
-public class SegmentButton: UIButton {
-    public static let borderColor: UIColor = .silver
-    public static let borderWidth = 1.5 as CGFloat
+class SegmentButton: UIButton {
+    static let borderColor: UIColor = .silver
+    static let borderWidth = 1.5 as CGFloat
```

Comment:
`borderWidth: CGFloat  = 1.5` is better.

Reply:
Consider it fixed 😉